database/proceedings/2019.json

```diff
@@ -0,0 +1,2588 @@
+[
+  {
+    "title": "Data Usage in MIR: History & Future Recommendations",
+    "author": [
+      "Wenqin Chen",
+      "Jessica Keast",
+      "Jordan Moody",
+      "Corinne Moriarty",
+      "Felicia Villalobos",
+      "Virtue Winter",
+      "Xueqi Zhang",
+      "Xuanqi Lyu",
+      "Elizabeth Freeman",
+      "Jessie Wang",
+      "Sherry Cai",
+      "Katherine Kinnaird"
+    ],
+    "year": "2019",
+    "doi": "10.5281/zenodo.3527733",
+    "url": "https://doi.org/10.5281/zenodo.3527733",
+    "ee": "https://zenodo.org/api/files/e3b5f4f4-981d-43f6-b9f8-02c1925702a8/ismir2019_paper_000001.pdf",
```

Comment:
These urls seem to no longer be valid, I'm not sure how the zenodo uploader script decided to use this value.
Our options are: 

 * Try and get a better direct url (https://zenodo.org/record/3527733 -> https://zenodo.org/record/3527733/files/ismir2019_paper_000001.pdf?download=1)
 * Link to archives.ismir.net

Reply:
I changed it back to archives.ismir.net

Reply:
Hm...what happened...actually all old ones are broken so should we move to archives.ismir in general?

Reply:
Maybe Zenodo changed their internal link generation. Only safe option is to use the DOI URL anyways.
Fine to delete "ee" in the future.